### ISSUE TITLE
[Bug][Github] Pipelines failing on converting issue to incident

### ISSUE THREAD

**Issue by sanjchhoker** [apache/devlake#8122]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

Pipelines only using github integration is failing consistently with below on dora step.
Checked database and we have nothing in incident table.

![IMG_8431](https://github.com/user-attachments/assets/a05e074a-bf91-4fe2-b327-b4de3ba7ac32)

### What do you expect to happen

Pipeline should run successfully

### How to reproduce

Run pipeline with a github connection.

### Version

v1.0.1

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by sanjchhoker** [comment]
@dosu what will then be the column type for type column in issues table?

**Comment by sanjchhoker** [comment]
@dosu this is what it is present then not sure what is the cause of failing pipeline

**Comment by sanjchhoker** [comment]
Resolved this by adding below columns
incident in issues
incident in issue_assignees
boards in project_mapping

**Comment by kgalli** [comment]
I ran into exactly this problem with version 1.0.1 using Postgres. I did not ran into this issue with MySQL. As I have to use Postgres in my environment can you provide some hints if this is a recent bug and if this is supposed to be fixed. From the previous comment I only get that certain columns are added to certain tables but I can not see any PR related which addresses this. 

See some part of the error message I get in the logs:

```
ERROR: column \"INCIDENT\" does not exist (SQLSTATE 42703)\n\x1b[0m\x1b[33m[0.210ms] \x1b[34;1m[rows:0]\x1b[0m \n\t\tDELETE\n\t\tFROM incidents
```

I'm happy to provide a PR I just need to understand if this is supposed to be already fixed or if the mentioned tables have to be patched. I also wonder why this is working on MySQL. Maybe there are some migrations to fix? Any hint would be much appreciated and I'm happy to support to get things fixed. I just need a starting point.

**Comment by vateseeb** [comment]
@kgalli I think the issue was fixed with this PR #8163 and it is included in the release [v1.0.2-beta4](https://github.com/apache/incubator-devlake/releases/tag/v1.0.2-beta4)
